compiler/rustc_mir_transform/src/check_alignment.rs

```diff
@@ -14,6 +14,10 @@ pub struct CheckAlignment;
 
 impl<'tcx> MirPass<'tcx> for CheckAlignment {
     fn is_enabled(&self, sess: &Session) -> bool {
+        // FIXME(#112480) MSVC and rustc disagree on minimum stack alignment on x86 Windows
+        if sess.target.llvm_target == "i686-pc-windows-msvc" {
```

Comment:
Is this potentially going to make some of the MIR tests for this fail on i686-msvc?  Are there any tests that should get something like a `ignore-i686-pc-windows-msvc`?

Reply:
I think just the test for the alignment check itself. The mir-opt tests have debug assertions disabled and I think I was otherwise pretty careful to modify the other tests so that they are not perturbed by them, but that was some time ago.

I'd prefer to be able to run the msvc test suite locally but I don't think that's supported on a Linux host?